src/main/java/com/google/cloud/tools/appengine/configuration/RunConfiguration.java

```diff
@@ -57,54 +39,18 @@ private RunConfiguration(
       List<Path> services,
       @Nullable String host,
       @Nullable Integer port,
-      @Nullable String adminHost,
```

Comment:
all this stuff was devserver v2 related?

Reply:
Is there a doc or anything where we can quickly compare and verify which options are for v1 and v2?

Reply:
There is in the code, parameters we were intentionally ignoring: https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/780/files#diff-f3a8b789822cdfedaed11b529b5fef47L210